src/routes/api/installation/api-installation-router.ts

```diff
@@ -7,25 +7,27 @@ import { ApiInstallationSyncPost } from "./api-installation-sync-post";
 import { ApiInstallationGet } from "./api-installation-get";
```

Comment:
I believe we also need to update all our runbooks?

Reply:
Ticket created ARC-1644.
For cloud, run book is same, so no changes.
For GHES, will do it in above ticket.

Reply:
Ok,sorry, I'm confused now :) Let me summarise my questions/concerns:

1. Could you please clarify who calls these endpoints?
2. Since we are prefixing it essentially with `githubapp` now, everything that was calling these endpoints will be broken now (no matter if it is in the code or in the runbooks), which is the cloud installations essentially (because GHES is not in prod yet)... What's the plan about it? Or noone is calling them now?

Reply:
No worries, let me try to explain:
1. I believe only human or machine call these apis. (LOL, I mean pollinator or anyone does some ops in prod)
2. There's a `?` in the line `ApiInstallationRouter.use(`(/githubapp/:gitHubAppId(\\d+))?`, subRouter);`. I think that does the trick, that it can accept with or without the `/githubapp/:gitHubAppId` in the route. If you look closer to the test, it should test the scenarios that api is hitting both with and without the new route.

So in short, for current runbook (pollinator, human), cloud supporting is unchanged. Only added support is GHES, therefore I think runbook for GHES can be done in another ticket. 

Let me know if you spot anything wrong in the routing still. Thanks.

Reply:
Gotcha @gxueatlassian , thanks for the explanation! It makes much more sense now